src/base.jl

```diff
@@ -938,34 +895,29 @@ Return a TimeSeriesData for a subset of the time series range stored for these p
 function get_time_series(
     ::Type{T},
     component::Component,
-    initial_time::Dates.DateTime,
-    label::AbstractString,
+    name::AbstractString,
```

Comment:
The get_time_series method in IS allows passing an initial time and horizon here.

Reply:
Agree. Must fix.